tests/scripts/all.sh

```diff
@@ -2710,20 +2711,37 @@ component_test_variable_ssl_in_out_buffer_len () {
 }
 
 component_test_variable_ssl_in_out_buffer_len_CID () {
-    msg "build: MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH and MBEDTLS_SSL_DTLS_CONNECTION_ID enabled (ASan build)"
-    scripts/config.py set MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH
+    msg "build: MBEDTLS_SSL_DTLS_CONNECTION_ID (standard) enabled (ASan build)"
```

Comment:
I think this entire component can be removed: standard `MBEDTLS_SSL_DTLS_CONNECTION_ID` is already enabled in the default config, as well as the full config, so it's already tested as part of other components based on those configs.